draft-ietf-rtcweb-jsep.xml

```diff
@@ -2668,6 +2698,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                   layers have been established. Otherwise, stop transmitting RTP media,
                   although RTCP should still be sent, as described in 
                   <xref target="RFC3264"/>, Section 5.1. </t>
+		<t>If RID values are specified, include the RID header
+		extension in the RTP streams, as indicated in <xref
+		target="I-D.ietf-mmusic-rid"></xref>, Section 4)
+		</t>
+		<t>If simulcast is specified, send simulcast as specified in
```

Comment:
I think this should be combined into the "prepare to start transmitting media" section above. You're just indicating how many RTP streams should be generated.